ros/src/foxglove_bridge/CMakeLists.txt

```diff
@@ -24,19 +26,14 @@ macro(enable_strict_compiler_warnings target)
   endif()
 endmacro()
 
-find_package(nlohmann_json QUIET)
 find_package(OpenSSL REQUIRED)
 find_package(Threads REQUIRED)
-find_package(websocketpp REQUIRED)
 find_package(ZLIB REQUIRED)
```

Comment:
we probably don't need these dependencies either? it should all be happening in our rust layer

Reply:
I'll defer paring these down to a later PR. Since we have a C++ WebSocket client used for testing, we may not be able to fully avoid removing these (other than ZLIB, which I confirmed isn't needed anymore). But in that PR, I'd like to refactor the build file to be a bit more granular and localize these deps to just test targets.